From: Andrew Shadura <andrewsh@debian.org>
Date: Sun, 12 Feb 2017 09:54:56 +0100
Subject: Use signedjson.sign instead of syutil.crypto.jsonsign

Functions from syutil.crypto.jsonsign are now available in
signedjson, so use that instead of depending on syutil.

Forwarded: yes
--
diff --git a/contrib/cmdclient/console.py b/contrib/cmdclient/console.py
index 8bb03ce..4918fa1 100755
--- a/contrib/cmdclient/console.py
+++ b/contrib/cmdclient/console.py
@@ -32,7 +32,7 @@ import urlparse
 import nacl.signing
 import nacl.encoding
 
-from syutil.crypto.jsonsign import verify_signed_json, SignatureVerifyException
+from signedjson.sign import verify_signed_json, SignatureVerifyException
 
 CONFIG_JSON = "cmdclient_config.json"
 
